packages/components/nodes/chains/SqlDatabaseChain/SqlDatabaseChain.ts

```diff
@@ -1,5 +1,5 @@
 import { ICommonObject, INode, INodeData, INodeParams } from '../../../src/Interface'
-import { SqlDatabaseChain, SqlDatabaseChainInput } from 'langchain/chains'
+import { SqlDatabaseChain, SqlDatabaseChainInput } from 'langchain/chains/sql_db'
```

Comment:
Had to make this change because of the newer langchainjs verrsion.

Refer PR: https://github.com/hwchase17/langchainjs/pull/1836/files